plugins/search-react/src/components/DefaultResultListItem/DefaultResultListItem.tsx

```diff
@@ -88,3 +99,25 @@ export const DefaultResultListItem = ({
     </Link>
   );
 };
+
+/**
+ * A higher order function providing AnalyticsContext to the DefaultResultListItemComponent.
```

Comment:
I wonder if this doc string actually gets exported out to consumers?  Could be good to put this within the component definition and instead provide a more useful comment here for consumers (or none).